lua/monome.lua

```diff
@@ -0,0 +1,23 @@
+-- monome device manager
+
+norns.monome = {}
+
+
+norns.monome.add = function(id, serial, name, dev)
+  -- TODO is there a better way to determine?
+  if (string.find(name, "arc") == nil) then
+    -- print("grid detected")
+    norns.grid.add(id,serial,name,dev)
+  else
+    -- print("arc detected")
+    norns.arc.add(id,serial,name,dev)
+  end
+end
+
+
+norns.monome.remove = function(id)
+  -- call both since we don't know if it's a grid or an arc
+  -- grid/arc functions will check for existence
```

Comment:
makes sense to check for monome device type here and call the appropriate remove handler (and remove the existence check from the handler then).

Reply:
same as above, i think i'll add `norns.arc.add` / `norns.arc.remove` instead - any objections?